apps/next/src/pages/Approve/ApproveDappConnection/ApproveDappConnection.tsx

```diff
@@ -77,23 +76,21 @@ export const ApproveDappConnection: FC = () => {
   }
 
   return (
-    <WalletTotalBalanceProvider>
```

Comment:
...and then let's leave it here (the only approval screen that actually shows all the wallets and their balances).